### ISSUE TITLE
Changes made in nested asset folder not being compiled

### ISSUE THREAD

**Issue by mklemme** [adunkman/connect-assets#281]
Seems like I am running into some issues with Sass (node-sass) and connect assets not compiling if I make changes to any file not directly in the parent folder.

Current folder structure:
    assets
    | - styles.sass
    | - framework
    | - - _settings.sass
    | - - _reset.sass
    | - - _base.sass
    | - - _colors.sass

Any changes I make to a file in the framework folder won't trigger the css file to be recompiled. I have to change the styles.sass file with extra `//` to trigger it. 

Possible bug?

**Comment by gabooh** [comment]
I have the same issue here with node-sass 1.2.3 and connect-assets 4.4.
I've tried to upgrade node-sass but then I get this problem https://github.com/adunkman/connect-assets/issues/289

**Comment by prcongithub** [comment]
Any solution to this? I am facing similar error on CentOS 6.5, node 5.4.0, npm 3.3.1.2,  connect-assets 5.0.1.
JS, CSS and Images in the assets/js, assets/css and assets/images folders are getting precompiled to public/assets but assets under any nested folders are not there in public/assets.
In app.js

```node
var asset_config = { paths: [ 'assets/js', 'assets/css', 'assets/images' ],
  buildDir: 'public/assets',
  _servePath: 'http://cdn-url.com/assets/' };
var assets = require('connect-assets')(asset_config);
app.use(assets);
```

**Comment by blakevanlan** [comment]
Sounds like this might have been fixed with #289. Let me know if that's not the case.
